app/javascript/packs/application.js

```diff
@@ -0,0 +1,21 @@
+/* eslint no-console:0 */
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+//
+// To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
+// layout file, like app/views/layouts/application.html.erb
+
+
+// Uncomment to copy all static images under ../images to the output folder and reference
+// them with the image_pack_tag helper in views (e.g <%= image_pack_tag 'rails.png' %>)
+// or the `imagePath` JavaScript helper below.
+//
+// const images = require.context('../images', true)
+// const imagePath = (name) => images(name, true)
+//= require jquery3
+//= require popper
+//= require bootstrap-sprockets
+
+console.log('Hello World from Webpacker')
```

Comment:
?

Reply:
It is default file generated by webpack. It is not our code. Should we remove this code? If we remove you will say that file is fully commented. So, should we remove this file? Then in comment below we remove another file named `hello_react.jsx`. In this case folder `packs` is empty. Should we remove this folder?

Reply:
We need this file, bootstrap won't work without these imports at least. I think that Dmitry was confused by using that sample print line. Deleted.